articles/cli.md

```diff
@@ -0,0 +1,178 @@
+## Bonsai Command-Line Interface (CLI)
+
+Bonsai CLI (Command-Line Interface) allows workflows to be launched from the OS command-line tool. In addition to opening workflows with the default graphic interface, the CLI opens the possibility to use other advanced features, that will be covered ahead, such as "no-editor" mode, initialization of variable's values, and specifying layouts.
+
+
+&nbsp;
+___
+### Installation and basic use
+Bonsai CLI can be called by targeting ```Bonsai.exe``` executable. To make sure you are able to call ```Bonsai``` from the command line:
+
+```cmd
+> pathToBonsaiExe\Bonsai.exe
+```
+Alternatively, by adding ```pathToBonsaiExe``` to your System Path, simply:
+
+```cmd
+> Bonsai
+```
+
+This command will open the default Bonsai editor.
```

Comment:
I assume this will only work after Bonsai has been added to the System Path variable, either by adding it using the command ```pathToBonsaiExe\Bonsai.exe``` or by adding it through the System Path settings menu?

Also, I assume this only works on Windows operating systems?

Reply:
The command ```pathToBonsaiExe\Bonsai.exe``` is simply a way to call bonsai without adding it to the system variables. It works by calling Bonsai.exe using its full path. Alternatively, you can add ```pathToBonsaiExe``` to the Path as a mentioned previously.

I think that, for now, we should probably stick with windows only. Some of this syntax is likely to change in Linux anyway.